roachtest: tpcc/multiregion/survive=region/chaos=true failed

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4099953&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4099953&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [bbd8658d09681135c30e07650798768ca0063f09](https://github.com/cockroachdb/cockroach/commits/bbd8658d09681135c30e07650798768ca0063f09):


```
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 636054.000000, to 636305.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (17) secondary error attachment
		  | error at from 2022-01-12T15:42:47Z, to 2022-01-12T15:47:18Z on metric workload_tpcc_newOrder_error_total{instance="34.75.122.71:2120"}: expected <=200 errors, found from 635816.000000, to 636032.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-01-12T15:42:47Z, to 2022-01-12T15:47:18Z on metric workload_tpcc_newOrder_error_total{instance="34.75.122.71:2120"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:256
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 635816.000000, to 636032.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (18) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (19) error at from 2022-01-12T15:37:38Z, to 2022-01-12T15:42:18Z on metric workload_tpcc_newOrder_error_total{instance="34.75.122.71:2111"}
		Wraps: (20) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1581
		Wraps: (21) expected <=200 errors, found from 89.000000, to 317.000000
		Error types: (1) *secondary.withSecondaryError (2) *secondary.withSecondaryError (3) *secondary.withSecondaryError (4) *secondary.withSecondaryError (5) *secondary.withSecondaryError (6) *secondary.withSecondaryError (7) *secondary.withSecondaryError (8) *secondary.withSecondaryError (9) *secondary.withSecondaryError (10) *secondary.withSecondaryError (11) *secondary.withSecondaryError (12) *secondary.withSecondaryError (13) *secondary.withSecondaryError (14) *secondary.withSecondaryError (15) *secondary.withSecondaryError (16) *secondary.withSecondaryError (17) *secondary.withSecondaryError (18) *withstack.withStack (19) *errutil.withPrefix (20) *withstack.withStack (21) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #73231 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot release-blocker]
</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-12241

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4690845&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4690845&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [834eaa0e83350486830867b5edd6e8809b52aa55](https://github.com/cockroachdb/cockroach/commits/834eaa0e83350486830867b5edd6e8809b52aa55):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:875: error at from 2022-03-27T13:23:10Z, to 2022-03-27T13:27:50Z on metric workload_tpcc_newOrder_error_total{instance="34.139.231.25:2122"}: expected <=200 errors, found from 0.000000, to 1101.000000
		(1) secondary error attachment
		  | error at from 2022-03-27T13:23:10Z, to 2022-03-27T13:27:50Z on metric workload_tpcc_payment_error_total{instance="34.139.231.25:2122"}: expected <=200 errors, found from 2.000000, to 942.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-03-27T13:23:10Z, to 2022-03-27T13:27:50Z on metric workload_tpcc_payment_error_total{instance="34.139.231.25:2122"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 2.000000, to 942.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-03-27T13:23:10Z, to 2022-03-27T13:27:50Z on metric workload_tpcc_newOrder_error_total{instance="34.139.231.25:2122"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 0.000000, to 1101.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820270&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4820270&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [9bff1de7115984957978416d829feb582eb2664e](https://github.com/cockroachdb/cockroach/commits/9bff1de7115984957978416d829feb582eb2664e):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:875: error at from 2022-04-07T13:31:49Z, to 2022-04-07T13:36:29Z on metric workload_tpcc_newOrder_error_total{instance="34.148.168.209:2121"}: expected <=200 errors, found from 0.000000, to 624.000000
		(1) secondary error attachment
		  | error at from 2022-04-07T13:31:49Z, to 2022-04-07T13:36:29Z on metric workload_tpcc_newOrder_error_total{instance="34.148.168.209:2112"}: expected <=200 errors, found from 0.000000, to 668.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-04-07T13:31:49Z, to 2022-04-07T13:36:29Z on metric workload_tpcc_newOrder_error_total{instance="34.148.168.209:2112"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 0.000000, to 668.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-04-07T13:31:49Z, to 2022-04-07T13:36:29Z on metric workload_tpcc_newOrder_error_total{instance="34.148.168.209:2121"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 0.000000, to 624.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4878150&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4878150&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [43457fc6e2f5324823cca4156d9799453753b9cc](https://github.com/cockroachdb/cockroach/commits/43457fc6e2f5324823cca4156d9799453753b9cc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:875: error at from 2022-04-13T12:57:59Z, to 2022-04-13T13:02:39Z on metric workload_tpcc_newOrder_error_total{instance="34.148.85.205:2122"}: expected <=200 errors, found from 35.000000, to 678.000000
		(1) secondary error attachment
		  | error at from 2022-04-13T12:57:59Z, to 2022-04-13T13:02:39Z on metric workload_tpcc_payment_error_total{instance="34.148.85.205:2122"}: expected <=200 errors, found from 0.000000, to 556.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-04-13T12:57:59Z, to 2022-04-13T13:02:39Z on metric workload_tpcc_payment_error_total{instance="34.148.85.205:2122"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 0.000000, to 556.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-04-13T12:57:59Z, to 2022-04-13T13:02:39Z on metric workload_tpcc_newOrder_error_total{instance="34.148.85.205:2122"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 35.000000, to 678.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4935109&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [7ea67fd5d4966508877fc88ba21bdd498ff162e8](https://github.com/cockroachdb/cockroach/commits/7ea67fd5d4966508877fc88ba21bdd498ff162e8):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:876: error at from 2022-04-20T12:49:35Z, to 2022-04-20T12:54:15Z on metric workload_tpcc_newOrder_error_total{instance="35.237.163.81:2111"}: expected <=200 errors, found from 26.000000, to 1755.000000
		(1) secondary error attachment
		  | error at from 2022-04-20T12:49:35Z, to 2022-04-20T12:54:15Z on metric workload_tpcc_payment_error_total{instance="35.237.163.81:2111"}: expected <=200 errors, found from 0.000000, to 1468.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-04-20T12:49:35Z, to 2022-04-20T12:54:15Z on metric workload_tpcc_payment_error_total{instance="35.237.163.81:2111"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 0.000000, to 1468.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-04-20T12:49:35Z, to 2022-04-20T12:54:15Z on metric workload_tpcc_newOrder_error_total{instance="35.237.163.81:2111"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 26.000000, to 1755.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139612&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5139612&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [1a91d7cb7b93dfef5dcaf872125875cefa3e0190](https://github.com/cockroachdb/cockroach/commits/1a91d7cb7b93dfef5dcaf872125875cefa3e0190):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:876: error at from 2022-05-10T13:42:59Z, to 2022-05-10T13:47:39Z on metric workload_tpcc_newOrder_error_total{instance="34.139.40.197:2122"}: expected <=200 errors, found from 605404.000000, to 607861.000000
		(1) secondary error attachment
		  | error at from 2022-05-10T13:42:59Z, to 2022-05-10T13:47:39Z on metric workload_tpcc_delivery_error_total{instance="34.139.40.197:2122"}: expected <=200 errors, found from 60544.000000, to 60798.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-05-10T13:42:59Z, to 2022-05-10T13:47:39Z on metric workload_tpcc_delivery_error_total{instance="34.139.40.197:2122"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 60544.000000, to 60798.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-05-10T13:42:59Z, to 2022-05-10T13:47:39Z on metric workload_tpcc_newOrder_error_total{instance="34.139.40.197:2122"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 605404.000000, to 607861.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80381 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1.0]
- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5148992&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5148992&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50](https://github.com/cockroachdb/cockroach/commits/bc1ee7c7c276984fce8ff5ba4fcfcdff335dde50):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:876: error at from 2022-05-12T12:43:55Z, to 2022-05-12T12:48:35Z on metric workload_tpcc_newOrder_error_total{instance="34.148.201.75:2122"}: expected <=200 errors, found from 0.000000, to 592.000000
		(1) secondary error attachment
		  | error at from 2022-05-12T12:43:55Z, to 2022-05-12T12:48:35Z on metric workload_tpcc_payment_error_total{instance="34.148.201.75:2122"}: expected <=200 errors, found from 0.000000, to 227.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2022-05-12T12:43:55Z, to 2022-05-12T12:48:35Z on metric workload_tpcc_payment_error_total{instance="34.148.201.75:2122"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  |   | runtime.goexit
		  |   | 	GOROOT/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=200 errors, found from 0.000000, to 227.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (3) error at from 2022-05-12T12:43:55Z, to 2022-05-12T12:48:35Z on metric workload_tpcc_newOrder_error_total{instance="34.148.201.75:2122"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=200 errors, found from 0.000000, to 592.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80381 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1.0]
- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5163863&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5163863&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b](https://github.com/cockroachdb/cockroach/commits/bc5f5b7159ed2b7dbffa94bb2a590fdddb642b9b):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:876: error at from 2022-05-15T13:39:07Z, to 2022-05-15T13:43:47Z on metric workload_tpcc_payment_error_total{instance="35.196.77.245:2111"}: expected <=200 errors, found from 405328.000000, to 405957.000000
		(1) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (2) error at from 2022-05-15T13:39:07Z, to 2022-05-15T13:43:47Z on metric workload_tpcc_payment_error_total{instance="35.196.77.245:2111"}
		Wraps: (3) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (4) expected <=200 errors, found from 405328.000000, to 405957.000000
		Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80381 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1.0]
- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190042&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=5190042&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [a35b848edb3f5562199b0f56bc27a6c576073d5d](https://github.com/cockroachdb/cockroach/commits/a35b848edb3f5562199b0f56bc27a6c576073d5d):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:267,tpcc.go:582,test_runner.go:876: error at from 2022-05-18T12:54:16Z, to 2022-05-18T12:58:56Z on metric workload_tpcc_newOrder_error_total{instance="35.185.127.202:2110"}: expected <=200 errors, found from 618054.000000, to 618640.000000
		(1) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:202
		  | [...repeated from below...]
		Wraps: (2) error at from 2022-05-18T12:54:16Z, to 2022-05-18T12:58:56Z on metric workload_tpcc_newOrder_error_total{instance="35.185.127.202:2110"}
		Wraps: (3) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:73
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:201
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:51
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:277
		  | runtime.goexit
		  | 	GOROOT/src/runtime/asm_amd64.s:1581
		Wraps: (4) expected <=200 errors, found from 618054.000000, to 618640.000000
		Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #80381 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1.0]
- #78619 roachtest: tpcc/multiregion/survive=region/chaos=true failed [C-test-failure O-roachtest O-robot branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/tpcc.go
+++ b/pkg/cmd/roachtest/tests/tpcc.go
@@ -614,6 +614,21 @@ func registerTPCC(r registry.Registry) {
 							if err != nil {
 								return tpccChaosEventProcessor{}, err
 							}
+							// We see a slow trickle of errors after a server has been force shutdown due
+							// to queries before the shutdown not fully completing. You can inspect this
+							// by looking at the workload logs and corresponding the errors with the
+							// prometheus graphs.
+							// The errors seen can be of the form:
+							// * ERROR: inbox communication error: rpc error: code = Canceled
+							//   desc = context canceled (SQLSTATE 58C01)
+							// Setting this allows some errors to occur.
+							allowedErrorsMultiplier := 5
+							if tc.survivalGoal == "region" {
+								// REGION failures last a bit longer after a region has gone down.
+								allowedErrorsMultiplier *= 20
+							}
+							maxErrorsDuringUptime := warehousesPerRegion * tpcc.NumWorkersPerWarehouse * allowedErrorsMultiplier
+
 							return tpccChaosEventProcessor{
 								workloadInstances: workloadInstances,
 								workloadNodeIP:    prometheusNodeIP[0],
@@ -624,17 +639,9 @@ func registerTPCC(r registry.Registry) {
 									"orderStatus",
 									"stockLevel",
 								},
-								ch:         chaosEventCh,
-								promClient: promv1.NewAPI(client),
-								// We see a slow trickle of errors after a server has been force shutdown due
-								// to queries before the shutdown not fully completing. You can inspect this
-								// by looking at the workload logs and corresponding the errors with the
-								// prometheus graphs.
-								// The errors seen can be be of the form:
-								// * ERROR: inbox communication error: rpc error: code = Canceled
-								//   desc = context canceled (SQLSTATE 58C01)
-								// Setting this allows some errors to occur.
-								maxErrorsDuringUptime: warehousesPerRegion * tpcc.NumWorkersPerWarehouse,
+								ch:                    chaosEventCh,
+								promClient:            promv1.NewAPI(client),
+								maxErrorsDuringUptime: maxErrorsDuringUptime,
 								// "delivery" does not trigger often.
 								allowZeroSuccessDuringUptime: true,
 							}, nil

